apps/docs/app/ui/main-nav.tsx

```diff
@@ -78,12 +78,27 @@ export const MainNav = ({ className }: MainNavProps) => {
   const routeApi = getRouteApi('/_docs');
   const { data } = routeApi.useLoaderData();
 
-  const componentsNavLinks = data.map((component) => ({
+  // Extract components and menu data
+  const components = data.components || [];
```

Comment:
Nullish coalescing operator her kanskje?

```suggestion
  const components = data.components ?? [];
```